.rubocop.yml

```diff
@@ -1,22 +1,3 @@
-AbcSize:
```

Comment:
I understand ChefStyle respects `.rubocop.yml` it's just confusing to end users. I would rather we disable the DuplicatedKey check in `spec/mixlib/versioning/versioning_spec.rb` using a comment:
https://timsjpark.wordpress.com/2015/10/10/disabling-select-cops-in-rubocop/